cli/docs/scorecard.md

```diff
@@ -26,7 +26,12 @@ This tutorial will walk you through setting up Scorecard for a single project.
     ```
     gsutil mb gs://$CAI_BUCKET_NAME
     ```
-5. Export the CAI data to GCS:
+5. Optionally, create a public GCS bucket to trigger a violation:
+    ```
+    gsutil mb gs://$PUBLIC_BUCKET_NAME
+    gsutil iam ch allUsers:objectViewer gs://$PUBLIC_BUCKET_NAME
+    ```
+6. Export the CAI data to GCS:
     ```
     # Export resource data
     gcloud asset export --output-path=gs://$CAI_BUCKET_NAME/resource_inventory.json \
```

Comment:
If we're reintroducing refresh can we skip this step?

Reply:
Updated to make this step optional (c9be8ea)

Refresh feature is only functional if the user runs scorecard as a service account. But the purpose of this particular guide and even the positioning of scorecard is mostly for ad-hoc runs and quick evaluations, likely on the user's local environment, hence I'd prefer not encourage user to download a service account key and keep the `gcloud asset export` option here because the gcloud command works with end user credential.

Reply:
Got it, makes sense.